app/routes/synonyms/synonyms.server.ts

```diff
@@ -72,7 +90,7 @@ export async function searchSynonymsByEventId({
     },
   } = await client.search({
     index: 'synonym-groups',
-    from: page && limit && (page - 1) * limit,
+    from: page * limit,
```

Comment:
Would you please explain this change to me?

Reply:
If you look at the circulars._archive._index/route file, you will see that in the search function it has `page - 1` passed into the search function. So that everything was the same whether it was synonym groups or not, I wanted to keep that as `page - 1` being passed in. If I didn't adjust it, it would be `page -1 -1` when it hit that line. The way that impacted things was that it would show the first page of results twice and you would not be able to see the last page of results.
If you look at this changed file https://github.com/nasa-gcn/gcn.nasa.gov/pull/2617/files#diff-e08b0c9f75242b7710fc6321f15b142e5694a47c82317853845577883ea2f6a9 you will see that I also had to adjust it there to bring it in line with this approach. 
it seemed better to keep it the same as circulars than to do something odd like page + 1 in the loader while leaving the page - 1 in the server function. This way they are all three the same.